azkaban-common/src/main/java/azkaban/executor/container/watch/AzPodStatusMetadata.java

```diff
@@ -110,7 +122,21 @@ public String getClusterName() {
         return Optional.empty();
       }
       executionId = executionLabel.substring(EXECUTION_ID_LABEL_PREFIX.length());
-      return Optional.of(new FlowPodMetadata(executionId, clusterName));
+
+      String cleanupDisabledLabel =
```

Comment:
It will be good to clean up this pod at sometime. Otherwise it will keep hanging there. Whether pod is 10 days old or some check. Please share your thoughts around this.

Reply:
Yes, it will be cleaned up as part of the 'stale-flow' check. Will add that to the ticket if it's not explicitly there.